modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/PhoneResourceTest.java

```diff
@@ -258,6 +268,23 @@ private long _getListTypeId(String listTypeId) {
 		return listType.getListTypeId();
 	}
 
+	private void _testPatchPhoneType() throws Exception {
+		Phone postPhone = testPatchPhone_addPhone();
+
+		postPhone.setPhoneType("business");
+
```

Comment:
you're not saving this value... 
question: which is the default if you don't pass any value to the addPhone? because in this case we can just add with the default and change/verify only the patched one (that should be different from the default)

Reply:
@smottal yes, it's using a random string by default, it's just leftover code I forgot to delete

Reply:
no, in the randomPhone we are not setting it and I think it takes it from 
```
private long _getListTypeId(String listTypeId) {
		List<ListType> listTypes = ListTypeServiceUtil.getListTypes(
			_user.getCompanyId(), listTypeId);

		ListType listType = listTypes.get(0);

		return listType.getListTypeId();
	}
	```
we need to be sure that this is not "personal" otherwise our patch is not valid (as we are using the same value)

Reply:
@smottal true, I remember incorrectly.